app/controllers/api/v1/accounts_controller.rb

```diff
@@ -1,24 +1,43 @@
 module Api
-	module V1
-		class AccountsController < ApplicationController
+  module V1
+    class AccountsController < ApplicationController
+      resource_description do
+        short "API for managing Accounts"
+      end
 
-# Create an account
-#Parameters: Email and UserName 
-		def create
-			@account = Account.new(account_params)
-			if @account.save
-				render json: {status: "SUCCESS", message: "Account created succesfully", data:@account},status: :ok
-			else
-				
-				render json: {status: "ERROR", message: "Account not created", data:@account.errors},status: :unprocessable_entity
-			end
+      api :POST, '/accounts', "Create an account" 
+      param :Email, String, :desc => "Email ID", :required => true
```

Comment:
Please go through ruby programming guidelines, use snake_case instead of camelCase for variable  and method names